tests/actions/draftTableAction.test.js

```diff
@@ -1,68 +1,65 @@
 import {
-  ADD_DRAFT_ORDER_SUCCESS,
-  DELETE_DRAFT_ORDER_SUCCESS,
-  DELETE_ALL_DRAFT_ORDERS_SUCCESS
+  ADD_DRAFT_DRUG_ORDER_SUCCESS,
+  DELETE_DRAFT_DRUG_ORDER_SUCCESS,
+  DELETE_ALL_DRAFT_DRUG_ORDERS_SUCCESS
 } from '../../app/js/actions/actionTypes';
 import {
   addDraftOrder,
   deleteDraftOrder,
-  deleteAllDraftOrders,
+  deleteAllDrugDraftOrders,
 } from '../../app/js/actions/draftTableAction';
 
 describe('Discontinue Order Actions', () => {
   beforeEach(() => moxios.install());
   afterEach(() => moxios.uninstall());
 
-  it('should dispatch a draft order successfuly', async (done) => {
+  it('should dispatch ADD_DRAFT_DRUG_ORDER_SUCCESS successfully', async (done) => {
```

Comment:
What is the logic behind changing from the message text to the constant ADD_DRAFT_DRUG_ORDER_SUCCESS?

Reply:
To describe what the it block does, which is to dispatch the `ADD_DRAFT_DRUG_ORDER_SUCCESS ` action.

Reply:
I do not think i understand you. Are you trying to say that ADD_DRAFT_DRUG_ORDER_SUCCESS describes more exactly than "a draft order successfuly"?

Reply:
Yes, that's what I am trying to say. It's an action and only make's sense to tell exactly what is been dispatched.
In this case `ADD_DRAFT_DRUG_ORDER_SUCCESS `.